snuba/snuba_migrations/metrics_summaries/0003_metrics_summaries_add_segment_id_duration_group_columns.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set_name = StorageSetKey.METRICS_SUMMARIES
+
+columns: List[Column[Modifiers]] = [
```

Comment:
Can we also add the `trace_id`?

Reply:
It's already in there https://github.com/getsentry/snuba/blob/master/snuba/snuba_migrations/metrics_summaries/0001_metrics_summaries_create_table.py#L18